loadtest/results/cross-vm-affinity-probe-2026-08-28/affinity_probe.py

```
@@ -0,0 +1,147 @@
+import base64
+import json
+import sys
+import time
+
+import grpc
+import exercise_pb2
+import exercise_pb2_grpc
+import urllib.request
+import urllib.error
+
+INTERNAL_TOKEN = "cross-vm-affinity-internal"
```

Ruff (0.16.2)

[error] Possible hardcoded password assigned to: "INTERNAL_TOKEN"

(S105)